cms/djangoapps/contentstore/views/course.py

```diff
@@ -642,6 +642,8 @@ def _create_or_rerun_course(request):
             fields['display_name'] = display_name
 
         if 'source_course_key' in request.json:
+            fields['advertised_start'] = None
```

Comment:
@adampalay that was also my concern, that this might be a too much of a ad-hoc solution. I'm not yet very familiar with the codebase, but following the code flow from the view to split, I couldn't find anything related to "resetting" specific fields.

So since in the view there is already a very specific field value assignment, I thought that was the best place to put the fix to at least get the discussion started!